typescript/apps/washboard-ui/src/app/components/lattice-settings.tsx

```diff
@@ -30,7 +30,14 @@ type LatticeFormInput = z.input<typeof formSchema>;
 type LatticeFormOutput = z.output<typeof formSchema>;
 
 export function LatticeSettings(): ReactElement {
-  const [{latticeUrl, latticeId, ctlTopicPrefix, retryCount}, setConfig] = useLatticeConfig();
+  const {config, setConfig} = useLatticeConfig();
+
+  const {latticeUrl, latticeId, ctlTopicPrefix, retryCount} = config ?? {
+    latticeUrl: '',
+    latticeId: 'default',
+    ctlTopicPrefix: 'wasmcloud.control',
+    retryCount: 5,
```

Comment:
Quick question: is there any particular reason why this is not `10` as [here](https://github.com/wasmCloud/wasmCloud/blob/9622399aeff74d2a79c3bed5980780fc57d98967/typescript/packages/lattice-client-core/src/lattice-client.ts#L43)?

Reply:
There is no need for the default value, I remove it.